src/components/10-atoms/radio/story.js

```diff
@@ -40,7 +40,7 @@ storiesOf('Components|Atoms/Radio', module)
 
     const template = html`
       <p>Knobs with a * only affect the first radio button</p>
-      <axa-fieldset horizontal>
+      <axa-fieldset horizontal nostretchonmobile>
```

Comment:
And if someone wants to see the "stretch" feature in action?! I would let these "foreign" story clean. You have this option in fieldset story, you dont need to add it here.

Reply:
Yes that's that's true, but then we could also argue if `horizontal` is in the right place here.

Reply:
Removed it, because it now works for all kinds of radios. You can use the knobs from fieldset to test ist. No demo necessary.

Reply:
You are right. I would also remove horizontal. These are `axa-fieldset` features and should be shown at fieldset story or demo.

Reply:
(outdated)